### DISCUSSION TITLE
Functionality to apply Custom XML Lighting Effects from AURA Creator

### DISCUSSION THREAD

**Discussion by ma7876** [seerge/g-helper#3489] [Ideas]
Hello, I really like G-Helper and would be happy to use it.

However,

I don't use default RBG effects from Armory Create, I use custom-made effects from AURA Creator. Of course, I understand it will be impossible to replace the same app, and I am not asking for it. But I would like to know if it is possible to apply already made saved effects which are XML files, using G-helper. Because I already have my effect ready, and I think if you guys were able to replicate all the effects from Armory Create, would be possible to have let's say button "Add XML file lighting effect" where I can my apply my custom effect.

This would be a game changer for me, and I will be happy to switch to G-Helper fully. I am not a programmer, so I am not sure how it all works, but it seems that it is possible. 

Thank you!

**Comment by seerge** [comment]
@ma7876 hello, Armoury and AURA are proprietary software :) Unfortunately I can't use any custom documents / files created there. 

G-Helper offers you access to all default (baked into firmware) backlight modes for laptop + couple of own (like Heatmap and Ambient).
